dask_expr/io/parquet.py

```diff
@@ -56,7 +53,7 @@ class ReadParquet(PartitionsFiltered, BlockwiseIO):
         "categories": None,
         "index": None,
         "storage_options": None,
-        "calculate_divisions": False,
+        "calculate_divisions": True,
```

Comment:
Is this asking for statistics by default?  If so, this has caused problems in the past.  Is it wise?  (genuine question, you know more than anyone here)

Reply:
Oops! This is actually a mistake. It is less common for metadata gathering to be painful now, but it is still nice to avoid it by default (when we know the files are sufficiently small, and are not calculating divisions or filtering).